client/components/post-card/search.jsx

```diff
@@ -39,10 +38,14 @@ function SearchByline( { post, site, feed } ) {
 
 export function SearchPostCard( { post, site, feed, onClick = noop, onCommentClick = noop } ) {
 	const featuredImage = post.canonical_image;
-	const hasPost = !! post;
+	const isPhotoOnly = post.display_type & DisplayTypes.PHOTO_ONLY;
+	const title = truncate( post.title, {
+		length: isPhotoOnly ? 50 : 140,
+		separator: /,? +/
```

Comment:
Seems like this will not work for non-whitespace langs? What happens if the title is 200 chars long? Like a URL?